packages/sdk/src/across/clients/bridgePool.ts

```diff
@@ -42,6 +42,7 @@ export type Pool = {
   blocksElapsed: number;
   secondsElapsed: number;
   liquidityUtilizationCurrent: string;
+  utilizedReserves: string;
```

Comment:
What is this changing? This seems like a bug fix?

Reply:
there was no bug, see resolution in [this thread](https://risklabs.slack.com/archives/C01RS73UX6K/p1643727262721569?thread_ts=1643688966.742979&cid=C01RS73UX6K).  The reason why i expose this now is to faciliate easier debugging if something like this happens again, this value was not being exposed, so it required going into sdk code, when it could easily be available as state on the FE.